docs/app/Examples/modules/Dropdown/common.js

```diff
@@ -337,7 +337,7 @@ export const countryOptions = [
   { key: 'zw', value: 'zw', flag: 'zw', text: 'Zimbabwe' },
 ]
 
-export const languageOptions = [
+export const languages = [
```

Comment:
For consistency with all the other exports in this file, let's restore this const name.